swift/Sources/Keystore/KeyStore.swift

```diff
@@ -35,6 +48,13 @@ public final class KeyStore {
         }
     }
 
+    @discardableResult
+    public func watch(coin: CoinType, address: String, xpub: String?) -> Watch {
```

Comment:
can we make it take an array of objects rather than just one? similar to wallets functionality

Reply:
Done